lib/WorldState.js

```diff
@@ -79,11 +78,12 @@ class WorldState {
    * @param {string} json The worldstate JSON string
    * @param {Object} [deps] The options object
    */
-  constructor(json, deps = defaultDeps) {
+  constructor(json, lang = 'en_US', deps = defaultDeps) {
```

Comment:
Just a note, also, the translations should also have at least the default translation with it's key, to make sure that it's fetchable if someone doesn't ever want to fall back to the default in the main object.